.eslintrc.yml

```diff
@@ -2,6 +2,15 @@ root: true
 
 plugins:
     - node
+    - prettier
 extends:
     - "eslint"
     - "plugin:node/recommended"
+    - "prettier"
+rules:
+    prettier/prettier:
+    - error
+    - tabWidth: 4
+      printWidth: 120
```

Comment:
I'm very happy to make it 80. The only reason I made it 120 was that I was trying to conform to what the code itself _seemed_ to be doing and making it 80 made the diff bigger. I'll change it and rebase the diff.

While we're at it, any reason to increase the default tab width by 100% 😉